.gitignore

```diff
@@ -1 +1,2 @@
 rest/models/
+.idea
```

Comment:
I'm of the opinion that you should probably have this in your personal global gitignore file, but at least add a newline here if u want to keep it.